src/registry/artifactory.rs

```diff
@@ -114,7 +115,7 @@ impl Artifactory {
             url.into()
         };
 
-        tracing::debug!("Hitting download URL: {artifact_url}");
+        debug!("Hitting download URL: {artifact_url}");
```

Comment:
I think these should all be properly formatted 🙈 (Not originated by you, but boy/girlscout rule of thumb; leave it better than you found it)